common/utils/src/main/resources/error/error-classes.json

```diff
@@ -3270,6 +3270,36 @@
     ],
     "sqlState" : "XXKST"
   },
+  "STV2_ENCODER_UNSUPPORTED_PREFIX_KEY" : {
+    "message" : [
+      "Encoder doesn't support prefix key for state store provider: <stateStoreEncoder>. Please use another state store provider that supports prefix keys, or omit the prefix key."
+    ],
+    "sqlState" : "00000"
+  },
+  "STV2_IMPLICIT_KEY_NOT_FOUND" : {
+    "message" : [
+      "Implicit key not found for operation on stateName=<stateName>"
+    ],
+    "sqlState" : "55019"
```

Comment:
Internal method called in stateStore. Object needs an implicit key defined in scope for methods to be called on it, thus 55019 applies well.